scripts/toss.coffee

```diff
@@ -15,7 +15,7 @@
 
 
 module.exports = (robot) ->
-  robot.hear /toss/i, (res) ->
+  robot.respond /toss/i, (res) ->
```

Comment:
Are you sure that this won't work for `bot I don't want a toss`?  or for `bot info alphatoss` ?